lib/iris/fileformats/grib/message.py

```diff
@@ -114,9 +115,19 @@ def sections(self):
         return self._raw_message.sections
 
     @property
+    def bmdi(self):
+        # Not sure of any cases where GRIB provides a fill value.
+        # Default for fill value is None.
```

Comment:
There is something to do with a fill value in the save rules (https://github.com/lbdreyer/iris/blob/4e4b9b0e48091e808503129cbcafd3a095a22e9b/lib/iris/fileformats/grib/_save_rules.py#L1198). To be honest, I don't really understand how it works or if it's something we should handle when loading.